shared/src/test/scala/mlscript/DiffTests.scala

```diff
@@ -483,13 +497,12 @@ class DiffTests
             
             if (mode.showParse)
               output(s"AST: $res")
+            
             val newMode = if (useIR) { mode.copy(useIR = true) } else mode
             val newNewMode = if (noTailRec) { newMode.copy(noTailRecOpt = true) } else newMode
+            val (postLines, nuRes) = postProcess(newNewMode, basePath, testName, res, output, raise)
+            postLines.foreach(output)
 
-            val (postLines, nuRes) = 
-              postProcess(newNewMode, basePath, testName, res, output, raise)
-            postLines.foreach(output)            
-            
```

Comment:
Please revert the needless whitespace changes. Why do you remove the empty-line separation between these lines and the previous block?